docs/guide/integrations-guide.md

```diff
@@ -99,38 +112,54 @@ After you install and activate an integration, you can:
+- [GoogleChat](../integration/alerts-googlechat-integration/)
+- [AlertOps](../integration/alerts-alertops-integration/)
+- [ServiceNow](../integration/alerts-servicenow-integration/)
 
 
 ## Creating New Integration
 
-Navigate to [Integrations Overview](https://apps.sematext.com/ui/integrations). Three different integrations are available:
+Go to the [Apps](https://apps.sematext.com/ui/integrations/apps) page to create a new Monitoring or Logs integration. Pick a solution type and all the available integrations for that solution will be listed. Hover over any integration card and click to create the respective App.
 
-- Log Shipping and Management App
-- Infrastructure and Application Performance Monitoring App
-- Notification Hooks through various third party integrations, email, and system defined custom webhooks
+Next step is to pick your environment that runs the service you want to monitor.
 
-Hover over any integration card and click to create respective App or a webhook. Initially, there will be no App status and count tag displayed in the left bottom corner of the card, but once related App(s) have been created tag will display the number of App, and grey and green tag colour are used to show if App are not or are receiving data respectively.
+![Sematext Cloud - Select Environment](../images/guide/integrations/environments.png)
 
-Image below illustrates an integration card being hovered over and the App status with the count tag displayed with one example App created.
-
-![Sematext Cloud - Create New App](https://sematext.com/docs/images/guide/integrations/add-new-logging-app.png "Sematext Cloud - Create New App")
+After you select your environment, agent installation page will show up which gives you step by step instructions on how to install [Sematext Agent](../agents/sematext-agent/).
 
 ### App Integration
 
 New integration creation userflow is the same for both monitoring as well as logging App. While Apache monitoring App is being created you can also create complementary logs App as well. It makes perfect sense, and it is the only way to have full observability into performance and issues of the Apache server for example, and that particular piece of a IT infrastructure puzzle.
 
-In other words, as monitoring App is being created you can create a logging App as well and vice versa. You can also invite other devops team members and provide them with the same tools and insights while using RBAC (role based access control) model to restrict or provide proper administrative user rights.
+In other words, as monitoring App is being created you can create a logging App as well and vice versa. 
+
+You can connect Logs and Monitoring Apps for quick access later when you are troubleshooting.
+
+You can also invite other devops team members and provide them with the same tools and insights while using RBAC (role based access control) model to restrict or provide proper administrative user rights.
+
+### Connected Apps
```

Comment:
@fulyauluturk JFYI we probably want to extract this into its own page with screenshots, explanation, linking to/from Split View, etc.

Reply:
Ticket created https://sematext.atlassian.net/browse/SC-14667